### ISSUE TITLE
Not able to cash in Scenarioquests

### ISSUE THREAD

**Issue by Tykros** [WarEmu/WarBugs#9114]
Hi, 
(This is my 1st bugreport, plz tell me if more info is needed)

On my Des Herder named Ghur I'm unable to cash-in quests from scenario's. 
As you can see in the 1st picture, no quests are flagged as done. While in the second you clearly see that is actually is marked as finished. (It's the same with all of them)
![bugrep scquest 1](https://cloud.githubusercontent.com/assets/25689339/22831538/ff1a8ac4-efaa-11e6-8ed4-3a5c03d83cb8.png)
![bugrep scquest 2](https://cloud.githubusercontent.com/assets/25689339/22831540/00f442d6-efab-11e6-919f-a7d6107ffdbc.png)

When I go to try and cash in the guest the quest-givers tells it's still in progress (aka, not finished)
tried so far without success:
 - abandon quest, retake, finish > no effect
 - relog/restart > no effect
 - by the end of the scenario go suicide for respawn > no effect

After that I logged an other char, played Reikland Factory and won.  screenshot after victory, walking toward quest-givers to check.
![bugrep scquest 3](https://cloud.githubusercontent.com/assets/25689339/22832135/2287272c-efad-11e6-8d6e-17c270f8e6eb.png) 

I don't run Tome or quest addons. 

Regards.
T.

**Comment by Razielhell** [comment]
Return in Inevitable city maybe?

**Comment by Luth-RoR** [comment]
> Changed all SC quests to have a shared drop off point in the capital cities, Order have Kaige Mortem in Altdorf Market Place and Destro have Riccekac Warshard close to the entrance of the Viper's Pit.

https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=19797
As Raziel wrote, did you try to finish the quest in the scenario or in the IC?

**Comment by Tykros** [comment]
Just noticed something else. I have Eternal citadel quest, not reikland factory (didn't have it when entering SC, probably overlooked at the time) Can't even take eternal citadel at T1 normaly.. :/

**Comment by Tykros** [comment]
Dropping quests in IC worked. Queing for Scenario's atm to doublecheck.

**Comment by Tykros** [comment]
Sitrep so far:
lost some scenario's or the SC's didn't pop-up agian to cash in until Reikland Factory popped. I came in the game by the end, des won. Took a screenshot but forgot it only works in windowed (which I did to make the Screenshots for the initial report), leaving it black. :/ 
Anyway, Quest was called correct, after we won, no bubbel above the head of the questgiver, book said quest fullfilled, questgivers said it was not. 
Respawned at Norsca where all new char start (could be because I was timing out because I was hanging around in the SC so long for making my failed screenshot).
Cashed in IC


Didn't check on the reikland factory - eternal citadel thingy on the other char yet.
